test/postoffice/messaging_test.exs

```diff
@@ -230,5 +230,23 @@ defmodule Postoffice.MessagingTest do
     test "count_failing_jobs/0 returns 0 if no retryable job exists" do
       assert Messaging.count_failing_jobs() == 0
     end
+
+    test "count_failing_jobs/0 returns failing job existents" do
+      Fixtures.create_failing_message(%{id: 1, user_id: 2})
+      Fixtures.create_failing_message(%{id: 2, user_id: 3})
+
+      assert Messaging.count_failing_jobs() == 2
+    end
+
+    test "count_failing_jobs/0 no returns retryable jobs when no exists" do
+      assert Messaging.get_failing_messages() == []
+    end
+
+    test "count_failing_jobs/0 returns retryable jobs" do
+      first_failing_job = Fixtures.create_failing_message(%{id: 1, user_id: 2})
+      second_failing_job = Fixtures.create_failing_message(%{id: 2, user_id: 3})
+
+      assert Messaging.get_failing_messages() == [first_failing_job, second_failing_job]
+    end
```

Comment:
I think the names of the tests are a bit misleading, since they are not testing `count_failing_jobs/0`
```suggestion
    test "get_failing_messages/0 returns no failing messages when they do not exist" do
      assert Messaging.get_failing_messages() == []
    end

    test "get_failing_messages/0 returns failing messages when they exist" do
      first_failing_job = Fixtures.create_failing_message(%{id: 1, user_id: 2})
      second_failing_job = Fixtures.create_failing_message(%{id: 2, user_id: 3})

      assert Messaging.get_failing_messages() == [first_failing_job, second_failing_job]
    end
```

I still see a bit of mix between the terms `jobs` and `messages`, so the suggestion may not be accurate. Please take this as a simple suggestion.